### PR TITLE
doc more duties for youtube admins

### PR DESCRIPTION
**Pull request by parispittman**
added more duties from my notes and what we do (or want to) now; very open to edits

### CODE REVIEW

File: communication/youtube-guidelines.md [hunk 1]
```diff
@@ -105,12 +105,19 @@ Conduct][coc]. Any potential violations should sent to <conduct@kubernetes.io>.
...
 editing. In other instances a Zoom recording may have significant dead-space
 leading the meeting itself. In these instances the playlist owners may be asked
 first to trim a recording; however there are times where this is inopportune and
-an admin may step in.
+an admin may step in. When in doubt, ask.
+
+### Housecleaning
+
+Random tasks that help promote a healthy channel:
+* Adding descriptions and/or keywords that would help the discoverability of information
```
Comment (mrbobbytables) [inline communication/youtube-guidelines.md]: tiny nit: use `-` instead `*` to keep with rest of doc / style guide :x

File: communication/youtube-guidelines.md [hunk 2]
```diff
@@ -105,12 +105,19 @@ Conduct][coc]. Any potential violations should sent to <conduct@kubernetes.io>.
...
-YouTube admins may be asked to help [trim] and [edit] recordings. Certain events
+YouTube admins should help [trim] and [edit] recordings. Certain events
 such as the Contributor Summits are not uploaded directly to YouTube and require
 editing. In other instances a Zoom recording may have significant dead-space
 leading the meeting itself. In these instances the playlist owners may be asked
 first to trim a recording; however there are times where this is inopportune and
-an admin may step in.
+an admin may step in. When in doubt, ask.
+
+### Housecleaning
```
Comment (mrbobbytables) [inline communication/youtube-guidelines.md]: Would you mind adding this to the ToC at the top?

File: communication/youtube-guidelines.md [hunk 3]
```diff
@@ -135,3 +142,4 @@ TODO: Add details on streaming/using OBS after  https://github.com/kubernetes/co
 [Community Meeting]: /events/community-meeting.md
 [Office Hours]: /events/office-hours.md
 [Meet our Contributors]: /mentoring/meet-our-contributors.md
+[moderation guidelines]: ./moderation.md
```
Comment (jeefy) [inline communication/youtube-guidelines.md]: Nit: Capitalize words (Moderation Guidelines)

**Comment by parispittman** [comment]
/close

im going to open up a fresh PR with more details at the moved file.

**Pull request closed without merging**